src/sentry/profiles/task.py

```diff
@@ -267,8 +266,9 @@ def _process_symbolicator_results(
 
 
 def _process_symbolicator_results_for_sample(profile: Profile, stacktraces: List[Any]) -> None:
-    if profile["platform"] == "rust":
-        for frame in stacktraces[0]["frames"]:
+    profile["profile"]["frames"] = stacktraces[0]["frames"]
+    if profile["platform"] in SHOULD_SYMBOLICATE:
```

Comment:
Why did this change from rust to include `cocoa`? This will never reach the `elif` below now and we'll be truncating the cocoa stack as if it was a rust stack.

Reply:
@Zylphrex because we want to remove `pre_context` and company from all the frames returned by symbolicator.

`iOS` like `Rust` might have it.

But the `if profile["platform"] == "rust"` wasn't supposed to be removed, only moved below. Thanks for catching that 👍🏻